cclient/api/core/attach_to_tangle.c

```diff
@@ -66,6 +67,10 @@ retcode_t iota_client_attach_to_tangle(iota_client_service_t const* const servic
         goto done;
       }
     }
+#else
+    log_info(client_core_logger_id, "[%s:%d] Local PoW is not supported on ESP32.\n", __func__, __LINE__);
```

Comment:
PoW might be absent not only for ESP32 but also for low-end devices such as Arm Cortex-M based SoC. Would it be better if there is a configurable switch and/or runtime info pointing PoW availability?

Reply:
it should depend on the platform. the user already has the choice of performing remote/local PoW at runtime.